addons/stock/models/stock_production_lot.py

```diff
@@ -93,3 +94,13 @@ def action_lot_open_quants(self):
         if self.user_has_groups('stock.group_stock_manager'):
             self = self.with_context(inventory_mode=True)
         return self.env['stock.quant']._get_quants_action()
+
+    @api.model
+    def _name_search(self, name, args=None, operator='ilike', limit=100, name_get_uid=None):
+        """ search name and reference """
+        args = args or []
+        domain = []
+        if name:
+            domain = ['|', ('name', operator, name), ('ref', operator, name)]
+        move_ids = self._search(expression.AND([domain, args]), limit=limit, access_rights_uid=name_get_uid)
```

Comment:
lot_ids

Reply:
@ryv-odoo did you ask lap we really want this feature ?